src/storage/db_storage.cpp

```diff
@@ -219,6 +219,18 @@ std::shared_ptr<DagBlock> DbStorage::getDagBlock(blk_hash_t const& hash) {
   return nullptr;
 }
 
+bool DbStorage::checkDagBlock(blk_hash_t const& hash) {
```

Comment:
We have transactionInDb and pbftBlockInDb methods, maybe dagBlockInDb would be a better name than checkDagBlock

Reply:
good point I will fix it